libgfortran/libgfortran.h

```diff
@@ -62,13 +62,19 @@ see the files COPYING3 and COPYING.RUNTIME respectively.  If not, see
 # include "quadmath_weak.h"
 #endif
 
-#ifdef __MINGW32__
+#ifdef __MINGW32__ && ! defined(__aarch64__)
```

Comment:
After my last findings, I've changed my mind and I'd revert this line change, otherwise, I can approve.